iocage_lib/ioc_start.py

```diff
@@ -1181,7 +1181,7 @@ def start_network_vnet_addr(self, iface, ip, defaultgw, ipv6=False):
         if ipv6:
             ifconfig = [iface, 'inet6', ip, 'up']
         else:
-            ifconfig = [iface, ip, 'up']
```

Comment:
We need to only alias on VNET when the initial IP has been set. We don't want them all to be aliases

Reply:
What do you mean, everything is an alias? There is no such thing as primary IP

Reply:
As far as I know there is a difference. Aliases have a broadcast on just their IP instead of the whole subnet.

Reply:
@skarekrow i haven't found anything which supports that ?
```
inet 192.168.0.26 netmask 0xffffff00 broadcast 192.168.0.255
net 192.168.0.27 netmask 0xffffff00 broadcast 192.168.0.255
```
Please advise. Also i am not sure what benefit we would have of having broadcast at all in that case ?